experimental/dds/tree2/src/feature-libraries/schemaSummarizer.ts

```diff
@@ -63,26 +64,38 @@ export class SchemaSummarizer implements Summarizable {
 		fullTree?: boolean,
 		trackState?: boolean,
 		telemetryContext?: ITelemetryContext,
+		incrementalSummaryContext?: IExperimentalIncrementalSummaryContext | undefined,
 	): ISummaryTreeWithStats {
-		// Currently no Fluid handles are used, so just use JSON.stringify.
-		const dataString = JSON.stringify(this.codec.encode(this.schema));
-		return createSingleBlobSummary(schemaStringKey, dataString);
+		const builder = new SummaryTreeBuilder();
+		if (
+			incrementalSummaryContext !== undefined &&
+			this.schemaIndexLastChangedSeq !== undefined &&
+			incrementalSummaryContext.latestSummarySequenceNumber >= this.schemaIndexLastChangedSeq
+		) {
+			builder.addHandle(
+				schemaStringKey,
+				SummaryType.Blob,
+				`${incrementalSummaryContext.summaryPath}/indexes/Schema/${schemaStringKey}`,
+			);
+		} else {
+			const dataString = JSON.stringify(this.codec.encode(this.schema));
+			builder.addBlob(schemaStringKey, dataString);
+		}
+		return builder.getSummaryTree();
 	}
 
 	public async summarize(
 		stringify: SummaryElementStringifier,
 		fullTree?: boolean,
 		trackState?: boolean,
 		telemetryContext?: ITelemetryContext,
+		incrementalSummaryContext?: IExperimentalIncrementalSummaryContext | undefined,
 	): Promise<ISummaryTreeWithStats> {
-		if (this.schemaBlobCache === undefined) {
-			// Currently no Fluid handles are used, so just use JSON.stringify.
-			const schemaText = JSON.stringify(this.codec.encode(this.schema));
-
-			// For now we are not chunking the the schema, but still put it in a reusable blob:
-			this.schemaBlobCache = await this.runtime.uploadBlob(IsoBuffer.from(schemaText));
-		}
-		return createSingleBlobSummary(schemaBlobKey, stringify(this.schemaBlobCache));
+		// Currently no Fluid handles are used, so just use JSON.stringify.
+		const dataString = JSON.stringify(this.codec.encode(this.schema));
```

Comment:
not a huge deal since `summarize` is going away somewhat soon, but would be nice to make this call into the same incremental logic OR make it throw to make it clear it's an unused codepath right now.

Reply:
updated to throw `summarize`